.github/workflows/setup-flatcar-sdk.sh

```diff
@@ -2,62 +2,51 @@
 
 set -euo pipefail
 
-sudo apt-get install -y lbzip2
-
-CORK_VERSION=$(curl -sL https://api.github.com/repos/kinvolk/mantle/releases/latest | jq -r .tag_name | sed -e 's/^v//')
-curl -L -o cork https://github.com/kinvolk/mantle/releases/download/v"${CORK_VERSION}"/cork-"${CORK_VERSION}"-amd64
-curl -L -o cork.sig https://github.com/kinvolk/mantle/releases/download/v"${CORK_VERSION}"/cork-"${CORK_VERSION}"-amd64.sig
-curl -LO https://kinvolk.io/flatcar-container-linux/security/image-signing-key/Flatcar_Image_Signing_Key.asc
-gpg --import Flatcar_Image_Signing_Key.asc
-gpg --verify cork.sig cork
-rm -f cork.sig Flatcar_Image_Signing_Key.asc
-chmod +x cork
-mkdir -p ~/.local/bin
-mv cork ~/.local/bin
-
-export PATH=$PATH:$HOME/.local/bin
-mkdir -p ~/flatcar-sdk
-
-pushd ~/flatcar-sdk || exit
-cork create || true
+sudo ln -sfn /bin/bash /bin/sh
+sudo apt-get install -y ca-certificates curl git gnupg lbzip2 lsb-release \
+    qemu-user-static
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg \
+    | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+echo \
+    "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] \
+    https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" \
+    | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
+sudo apt-get install -y docker-ce docker-ce-cli containerd.io \
+    docker-compose-plugin
 
-sudo tee "./chroot/etc/portage/make.conf" <<EOF
-PORTDIR="/mnt/host/source/src/third_party/portage-stable"
-PORTDIR_OVERLAY="/mnt/host/source/src/third_party/coreos-overlay"
-DISTDIR="/mnt/host/source/.cache/distfiles"
-PKGDIR="/var/lib/portage/pkgs"
-PORT_LOGDIR="/var/log/portage"
-EOF
+mkdir -p ~/flatcar-sdk
+git -C ~/flatcar-sdk clone https://github.com/flatcar-linux/scripts
 
-sudo tee "./chroot/etc/portage/repos.conf/coreos.conf" <<EOF
-[DEFAULT]
-main-repo = portage-stable
+pushd ~/flatcar-sdk/scripts || exit
 
-[gentoo]
-disabled = true
+source ci-automation/ci_automation_common.sh
+source sdk_container/.repo/manifests/version.txt
 
-[coreos]
-location = /mnt/host/source/src/third_party/coreos-overlay
+git submodule update --init --recursive
 
-[portage-stable]
-location = /mnt/host/source/src/third_party/portage-stable
-EOF
+arch="amd64"
+channel_version="alpha-${FLATCAR_VERSION_ID}"
+check_version_string "${channel_version}"
 
-# /var under the chroot has to be writable by the runner user
-sudo chown -R runner:docker ~/flatcar-sdk/chroot/var
+export SDK_NAME="flatcar-sdk-${arch}"
 
-function enter() ( exec cork enter -- $@ )
+# Pin the docker image version to that of the latest release.
+docker_sdk_vernum="$(curl -s -S -f -L \
+    https://raw.githubusercontent.com/flatcar-linux/manifest/flatcar-master/version.txt \
```

Comment:
The manifest repo won't be maintained any more, we should use the release server's version.txt.

Reply:
Right, done.